ckanext/authz_service/tests/test_actions.py

```diff
@@ -110,6 +110,9 @@ def test_authorize_request_resource_write_anon_user(self):
 @pytest.mark.usefixtures('with_plugins')
 class TestPublicKeyAction():
 
+    @pytest.mark.skipif(
+        toolkit.check_ckan_version(min_version='2.9'),
+        reason='Test fixture not supported in Python 3')
```

Comment:
The fixture is the custom extension one: [temporary_file(...)](https://github.com/datopian/ckanext-authz-service/blob/4d652a77244c26dfd87b4e2cf4276d61aa6f5d48/ckanext/authz_service/tests/__init__.py#L34), since Python 2 and 3 writes and reads files differently I couldn't came up with an easy implementation of this that runs on both pipelines.

It returns `True` if the CKAN version is greater than or equal to `min_version`, so It will skip if greater or equal than `2.9`